src/uu/users/Cargo.toml

```diff
@@ -16,6 +16,7 @@ path = "src/users.rs"
 
 [dependencies]
 clap = { workspace = true }
+utmp-classic = {version = "0.1.5"}
```

Comment:
please define it in /Cargo.toml and use `workspace = true`
and make it openbsd only